### PR TITLE
moved functions and control statements to lesson 3

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

**Comment by ayan-usgs** [comment]
@lindsaycarr,

Can you take a look at this PR when you have a moment?

File: vignettes/C_Clean.Rmd [hunk 1]
```diff
@@ -1,324 +1,531 @@
...
+
+```{r Exercise2, echo=FALSE}
+```
+
+##Exercise 2
+In this exercise we are going to practice merging our Gages data. You will remember that we have two datasets, site information and basin landcover types. We selected some rows out of the site data (based on state) but didn't select out of the basin landcover table, so we have two data frames, with differing numbers of rows and unknown order. Use your stickies!
+
+1. This is the only task we have for this exercise.  Add to your script a line (or more if you need it) to create a new data frame, `gages_data`, that is a merge of `gages_sites_nm` and `gages_cover_subset`, but with only lines in `gages_sites_nm` preserved in the output.  The column to merge on is "STAID"
+2. This data frame may have some `NA` values.  For the purposes of these lessons, it is better to remove these.  Add another line to your code and create a data frame that removes all NA values from `gages_data`.
+2. If that goes quickly, feel free to explore `rbind()` some.
```
Comment (lindsaycarr) [inline vignettes/C_Clean.Rmd]: there are two "2" steps

**Comment by lindsaycarr** [comment]
:+1:

**Comment by ayan-usgs** [comment]
I'll fix that typo. 

Also, might also make a pass at fixing the line endings before merging.

**Comment by lindsaycarr** [comment]
do you have write access to merge?

**Comment by ayan-usgs** [comment]
Nope. I can't do the merge.

Also, the line ending fix worked! :smile:

**Comment by lindsaycarr** [comment]
Yeah, I can't either. Can @jread-usgs merge? 

Awesome - much easier to see what was changed now :)

**Comment by lindsaycarr** [comment]
:+1: guess you got write access??

**Comment by jread-usgs** [comment]
:birthday:

**Comment by jread-usgs** [comment]
@lindsaycarr you have write access to this repo as well now.

**Comment by lindsaycarr** [comment]
thanks!

**Pull request merged**